### PR TITLE
chore: add v1 proxy tests for MSL

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## What are the changes introduced in this PR?

>add v1 proxy tests for MSL
New scenarios added = 1

## What is the related Linear task?

Resolves INT-1489

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3130/test-report.html

File: test/integrations/destinations/marketo_static_list/dataDelivery/other.ts [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+            ],
+            statTags: {
+              errorCategory: 'network',
+              errorType: 'retryable',
+              destType: 'MARKETO_STATIC_LIST',
+              module: 'destination',
+              implementation: 'native',
+              feature: 'dataDelivery',
+              destinationId: 'default-destinationId',
+              workspaceId: 'default-workspaceId',
```
Comment (utsabc) [inline test/integrations/destinations/marketo_static_list/dataDelivery/other.ts:55]: Can deduplicate these

File: test/integrations/destinations/marketo_static_list/dataDelivery/other.ts [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+                  attemptNum: 1,
+                  userId: 'default-userId',
+                  destinationId: 'default-destinationId',
+                  workspaceId: 'default-workspaceId',
+                  sourceId: 'default-sourceId',
+                  secret: {
+                    accessToken: 'default-accessToken',
+                  },
+                  dontBatch: false,
+                },
```
Comment (utsabc) [inline test/integrations/destinations/marketo_static_list/dataDelivery/other.ts:44]: Use the generate metadata fxn

File: test/integrations/destinations/marketo_static_list/dataDelivery/other.ts [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
```
Comment (utsabc) [inline test/integrations/destinations/marketo_static_list/dataDelivery/other.ts:101]: Same here

File: test/integrations/destinations/marketo_static_list/dataDelivery/other.ts [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+            statTags: {
+              errorCategory: 'network',
+              errorType: 'retryable',
+              destType: 'MARKETO_STATIC_LIST',
+              module: 'destination',
+              implementation: 'native',
+              feature: 'dataDelivery',
+              destinationId: 'default-destinationId',
+              workspaceId: 'default-workspaceId',
+            },
```
Comment (utsabc) [inline test/integrations/destinations/marketo_static_list/dataDelivery/other.ts:113]: replicated

File: test/integrations/destinations/marketo_static_list/dataDelivery/other.ts [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+                  sourceId: 'default-sourceId',
+                  secret: {
+                    accessToken: 'default-accessToken',
+                  },
+                  dontBatch: false,
+                },
+              },
+            ],
+            statTags: {
+              errorCategory: 'network',
```
Comment (utsabc) [inline test/integrations/destinations/marketo_static_list/dataDelivery/other.ts:277]: Lets deduplicate the `statTags` and `metadata`

**Review by utsabc — CHANGES_REQUESTED** [review]

**Review by utsabc — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Pull request merged**